sage/handlers/huggingface/inference.py

```diff
@@ -0,0 +1,28 @@
+from transformers import AutoModel, AutoTokenizer
+import logging
+import json
+
+
+def model_fn(model_dir):
+    return AutoModel.from_pretrained(model_dir, trust_remote_code=True)
+
+
+def input_fn(data, content_type="application/json"):
+    return data
+
+
+def predict_fn(data, model):
```

Comment:
this handler is a bit specific to the bertmesh model so maybe this is how we should name it